### ISSUE TITLE
Fetch additional content for threads from other servers

### ISSUE THREAD

**Issue by nikclayton** [pachli/pachli-android#65]

**Comment by kallekn** [comment]
Yes please!

**Comment by d3d9** [comment]
I was trying to see whether there is an open issue regarding the loading of additional posts in threads, is this one it?
This seems to be done by the server since Mastodon 4.5 if I understand correctly? <https://blog.joinmastodon.org/2025/11/mastodon-4.5/#fetch-all-replies-completing-the-conversation-flow>
When I view a thread in Pachli I sometimes notice there are more posts after I reload it.
Basically what's missing would just be the equivalent of the "More replies found" popup of the Mastodon UI, or some other, maybe automatic method that makes sure users actually become aware of / see the additional replies.
Please let me know if this needs a separate issue or if it already exists.

**Comment by d3d9** [comment]
Adding to my previous comment; I just realized, when you use the refresh button in the top right of the view of the post / replies, it already keeps your current position, that's great. So maybe this loading could actually become an automatic thing? Poll whether the server reports that additional replies are available, automatically trigger the refresh if that's the case, maybe show a toast that tells the user about that, without any confirmation/click needed.
Maybe some sort of slight highlight of the replies that were just added could also be useful.
